bindings/go/credentials/spec/config/v1/direct_credentials.go

```diff
@@ -18,3 +18,9 @@ type DirectCredentials struct {
 	Type       runtime.Type      `json:"type"`
 	Properties map[string]string `json:"properties"`
 }
+
+func CredentialsFromMap(properties map[string]string) *DirectCredentials {
+	return &DirectCredentials{
+		Properties: properties,
+	}
+}
```

Comment:
This never sets the type. Is that correct? Just want to make sure it's defaulted somewhere else, or something like that. :)